spec/aws/session_store/dynamo_db/locking/threaded_sessions_spec.rb

```diff
@@ -1,96 +0,0 @@
-# frozen_string_literal: true
-
-# Copyright 2013 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"). You
-# may not use this file except in compliance with the License. A copy of
-# the License is located at
-#
-#     http://aws.amazon.com/apache2.0/
-#
-# or in the "license" file accompanying this file. This file is
-# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
-# ANY KIND, either express or implied. See the License for the specific
-# language governing permissions and limitations under the License.
-
-require 'spec_helper'
-
-describe Aws::SessionStore::DynamoDB::RackMiddleware do
```

Comment:
Why are we removing this file? Should we be testing threaded sessions?

Reply:
I reverted it and fix to pass the test.
b7121961af1fe7cc6bff01a89491aed5eb9a5ef3